code/game/objects/items/weapons/grenades/ghettobomb.dm

```diff
@@ -59,7 +59,7 @@
 
 /obj/item/weapon/grenade/iedcasing/prime() //Blowing that can up
 	update_mob()
-	explosion(src.loc,-1,-1,-1, flame_range = range)	// no explosive damage, only a large fireball.

```

Comment:
Why didn't you just change the range var instead of removing its only use in the code?